src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed,
+  })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {typeof todo.user === 'object' && (
```

Comment:
```suggestion
    {todo.user && (
```
just check on todo.user